src/engines/Cesium/Feature/Resource/utils.ts

```diff
@@ -318,7 +318,7 @@ export const attachStyle = (
   if (entity.polygon) {
     const entityPosition = entity.position?.getValue(currentTime);
     const hierarchy = entity.polygon?.hierarchy?.getValue(currentTime) as PolygonHierarchy;
-    const coordinates: any = hierarchy?.positions?.map(position => [
+    const coordinates = hierarchy?.positions?.map(position => [
```

Comment:
Could you update here(includes point and polyline) to use getGeometryFromEntity?